resources/benchmark.ts

```diff
@@ -122,9 +122,12 @@ function collectSamples(modulePath: string) {
 
     if (sample.involuntaryContextSwitches > 0) {
       numOfConsequentlyRejectedSamples++;
-      if (numOfConsequentlyRejectedSamples > 5) {
-        throw Error(
-          'Can not sample benchmark due to 5 consequent runs beings rejected because of context switching',
+      if (numOfConsequentlyRejectedSamples === 5) {
+        console.error(
+          yellow(
+            '  Five or more consequent runs beings rejected because of context switching.\n' +
+            '  Measurement can take a significantly longer time and its correctness can also be impacted.',
```

Comment:
Question: if we still throw out bad samples, why is measurement correctness impacted when it finally completes?

Reply:
I guess you're just saying you don't know for sure, and that's fine.

Reply:
Yes, exactly. 
If you running a heavy background process it's causing all other kinds of issues, e.g. your CPU can thermal throttle (relevant only for local runs), shared caches, and memory access is affected.
This notice here is basically to say if we running on a PC with a heavy load we shouldn't try to figure out how it affects measurements. 
You simply should solve the root cause, which is not yet possible on our CI.